tests/migration/Core/V6_6/Migration1701776095FixDefaultMailFooterTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\Tests\Core\Migration\V6_6;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Migration\V6_6\Migration1701776095FixDefaultMailFooter;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Migration\V6_6\Migration1701776095FixDefaultMailFooter
+ */
+class Migration1701776095FixDefaultMailFooterTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    public function testCreationTimestamp(): void
```

Comment:
also please add a test case that the german mail template does not include the typo anymore after you ran your migration